index.html

```diff
@@ -283,14 +283,12 @@ <h2 class="banded" id="dataset">Dataset</h2>
 
     <h2>Reference</h2>
     <p>Please cite our paper if you use our code, data, model or results.</p>
-    <pre class="reference">@article{stevens2023bioclip,
-  title={BioCLIP: A Vision Foundation Model for the Tree of Life}, 
-  author={Samuel Stevens and Jiaman Wu and Matthew J Thompson and Elizabeth G Campolongo and Chan Hee Song and David Edward Carlyn and Li Dong and Wasila M Dahdul and Charles Stewart and Tanya Berger-Wolf and Wei-Lun Chao and Yu Su},
-  year={2023},
-  eprint={2311.18803},
-  archivePrefix={arXiv},
-  primaryClass={cs.CV}
-}</pre>
+    <pre class="reference">@inproceedings{stevens2023bioclip,
```

Comment:
```suggestion
    <pre class="reference">@inproceedings{stevens2024bioclip,
```
Since it's 2024 publication.